src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,41 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+        Manufacturer manufacturer = new Manufacturer("Honda", "Japan");
+        Driver driver = new Driver("Bob", "12345");
+        manufacturerService.create(manufacturer);
+        driverService.create(driver);
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        manufacturers.forEach(System.out::println);
+        List<Driver> drivers = driverService.getAll();
+        drivers.forEach(System.out::println);
+        Manufacturer updatedManufacturer = new Manufacturer("Toyota", "Japan");
+        updatedManufacturer.setId(1L);
```

Comment:
we can update and delete entities without setting ID. how can we do that?

Reply:
fixed